pkg/cert/cert.go

```diff
@@ -0,0 +1,80 @@
+/*
+Copyright 2024 The Kubeflow Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cert
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	cert "github.com/open-policy-agent/cert-controller/pkg/rotator"
+	"k8s.io/apimachinery/pkg/types"
+	ctrl "sigs.k8s.io/controller-runtime"
+)
+
+const (
+	certDir                  = "/tmp/k8s-webhook-server/serving-certs"
```

Comment:
This is the default value for the controller-runtime (https://github.com/kubernetes-sigs/controller-runtime/blob/76d3d0826fa9dca267c70c68c706f6de40084043/pkg/webhook/server.go#L144).
So, I prefer to use the default value. WDYT?

Reply:
@tenzen-y Do we need to set the cert dir if Controller Runtime has the default value for it ?

Reply:
@andreyvelich If we want to use the `/tmp/cert`, we need to set the value here: https://github.com/kubeflow/training-operator/blob/c20422067e3ef81df39d03c6f285353344d8f77d/cmd/training-operator.v1/main.go#L127-L129

Reply:
We need to set the `/tmp/k8s-webhook-server/serving-certs` to the `CertRotator` since the `CertRotator` is not part of controller-runtime project. The `/tmp/k8s-webhook-server/serving-certs` is the default value only in the controller-runtime project.

Reply:
You are right, thanks for the clarification!